inputs/manual_input.csv

```diff
@@ -318,12 +318,32 @@ offwind-float,FOM,2050,1.39,%/year,2020,https://doi.org/10.1016/j.adapen.2021.10
 organic rankine cycle,FOM,2020,2,%/year,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551","Both for flash, binary and ORC plants. See Supplemental Material for details"
-organic rankine cycle,investment,2020,1500,USD/kW_el,2020,Tartiere and Astolfi 2017: A world overview of the organic Rankine cycle market,"Low rollout complicates the estimation, compounded by a dependence both on plant size and temperature."
-geothermal,CO2 intensity,2020,0.12,tCO2/MWh_el,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551",Likely to be improved; Average of 85 percent of global egs power plant capacity
+organic rankine cycle,investment,2020,1376,EUR/kW_el,2023,Tartiere and Astolfi 2017: A world overview of the organic Rankine cycle market,"Low rollout complicates the estimation, compounded by a dependence both on plant size and temperature, converted from 1500 USD/kW using currency conversion 1.09 USD = 1 EUR."
+geothermal,CO2 intensity,2020,0.12,tCO2/MWh_el,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551",Likely to be improved; Average of 85 percent of global egs power plant capacity; Result of fluid circulation through rock formations
 geothermal,lifetime,2020,30,years,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551",
-geothermal,efficiency central heat,2020,0.8,per unit,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551; Breede et al. 2015: Overcoming challenges in the classification of deep geothermal potential, https://eprints.gla.ac.uk/169585/","This is a rough estimate, depends on local conditions"
+geothermal,district heat-input,2020,0.8,MWh_thdh/MWh_th,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551; Breede et al. 2015: Overcoming challenges in the classification of deep geothermal potential, https://eprints.gla.ac.uk/169585/","Heat-input, District Heat-output. This is an assessment of typical heat losses when heat is transmitted from the EGS plant to the DH network, This is a rough estimate, depends on local conditions"
```

Comment:
Question: Shouldn't heat input vs. heat output be > 1?

Based on the previous 0.8 efficiency value like: 1 / 0.8 = 1.25 ?

if not, maybe I'm misunderstanding the value.

Reply:
I see where the intuition is coming from, but I was convinced otherwise, and feel confident in 80% 👍

Reply:
Ok... I guess? It doesn't make any sense to me and the description in the file isn't explaining it. Would you care to indulge me? Or should I just accept and be quiet? ;-)

Reply:
Heya @euronion, sorry for missing this and for going 'dude, trust me.'
I hope I get this right, but I think your intuition is summarized as such:
Say we have `Q_in` with temperature 150 degreesC, and it meets demand `Q_out` with a temperature of 90 degrees, then shouldnt that happen at an efficiency of > 1.. Is that the intuition? Otherwise we are talking about different things.
What this intuition misses is that the temperature is already priced into the power (roughly mass flow times temperature). So when thinking about situation where `Q_in = Q_out`, but they have a 60 degreeC temperature difference what we are really talking about is the hotter flow having a much smaller mass flow and us mixing in cooler liquid to accommodate the properties required at `Q_out`... Does that help? Maybe it wasnt the problem, does not feel very complex but mass flow is what was missing in my intuition.